src/components/Textbook/PageNumber.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Button from '../Shared/Buttons';
+import Styles from './textbook.module.scss';
+const PageNumber = () => {
+  return (
+    <div className={Styles.pageNumber}>
+      <Button outline={true} color={'cyan'} label="←" size="round" />
+      <Button outline={true} color={'cyan'} label="1" size="round" />
+      <Button outline={false} color={'cyanDark'} label="2" size="round" />
+      <Button outline={true} color={'cyan'} label="3" size="round" />
+      <p className="paragrahp--p1">...</p>
```

Comment:
![image](https://github.com/taodemy/taotalk/assets/76163407/0ed289b0-7c1a-4e8b-80b2-5b63b0795583)
the vertical direction of this point needs to be aligned to the bottom